packages/core/src/views/shared/Modal.svelte

```diff
@@ -65,11 +72,16 @@
 </style>
 
 <section class="fixed" transition:fade>
-  <div on:click={close} class="background flex justify-center items-center">
-    <div on:click|stopPropagation class="flex relative max-height">
-      <div class="modal-overflow relative flex justify-center">
-        <div class="modal relative">
-          <slot />
+  <div
+    on:click={close}
+    class="background flex items-center justify-center relative"
+  >
+    <div class="flex modal-position">
```

Comment:
```suggestion
    <div class="flex modal-position absolute">
```

Reply:
This needs to be positioned absolutely so that the positioning variables work.

Reply:
Good catch!